packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -241,19 +241,75 @@ const start = async zcf => {
       });
       ({ priceAuthority, adminFacet: priceAuthorityAdmin } = paKit);
     },
-    async initOracle(oracleInstance, query = undefined) {
+    makeOracleInvitation: async () => {
```

Comment:
My understanding of this is a little fuzzy. There a unit test that tells a little more about how it works?

_wishing for that widget that adds test coverage details to each PR_

Or do you feel like adding a paragraph of JSDoc?